src/installer/pkg/sfx/installers/Directory.Build.props

```diff
@@ -0,0 +1,8 @@
+<Project>
+  <Import Project="$([MSBuild]::GetPathOfFileAbove(Directory.Build.props, $(MSBuildThisFileDirectory)..))" />
+  
+  <PropertyGroup>
+    <BuildDebPackage Condition="'$(TargetsLinuxGlibc)' == 'true'">true</BuildDebPackage>
+    <BuildRpmPackage Condition="'$(TargetsLinuxGlibc)' == 'true'">true</BuildRpmPackage>
```

Comment:
These are set in .proj files, do we also need them in .props file?

Reply:
Yes. This provides the defaults for the installers, but the runtime-deps installers only build some of the flavors (deb or rpm, not both).